### PR TITLE
🌐 Update Spanish translation including new illustrations in `docs/es/docs/async.md`

### PR DESCRIPTION
**Pull request by andresbermeoq**
The changes that were explicated in the #9474.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ archive.zip
 # vim temporary files
 *~
 .*.sw?
+.python-version
```
Comment (LaurEars) [inline .gitignore:26]: This might be a requirement only for your local environment. Not sure it's necessary for the whole project.

**Review by LaurEars — CHANGES_REQUESTED** [review]
- One question about changes to `.gitignore` - pls change ❌
- Ran this branch locally and images load properly and in correct locations, according to the original english documentation ✅

File: .gitignore (hunk 1 above)
Comment (tiangolo) [reply .gitignore:26]: Agreed 🤓

Can you please remove it @andresbermeoq? :pray:

File: docs/es/docs/async.md [hunk 2]
```diff
@@ -104,24 +104,40 @@ Para entender las diferencias, imagina la siguiente historia sobre hamburguesas:
...
 Llega tu turno, haces tu pedido de 2 hamburguesas impresionantes para esa persona 😍 y para ti.
 
-Pagas 💸.
+<img src="https://fastapi.tiangolo.com/img/async/concurrent-burgers/concurrent-burgers-02.png" alt="illustration">
 
 El cajero 💁 le dice algo al chico de la cocina 👨‍🍳 para que sepa que tiene que preparar tus hamburguesas 🍔 (a pesar de que actualmente está preparando las de los clientes anteriores).
 
+<img src="https://fastapi.tiangolo.com/img/async/concurrent-burgers/concurrent-burgers-03.png" alt="illustration">
+
+Pagas 💸.
```
Comment (realFranco) [inline docs/es/docs/async.md:117]: Hello 👋, thank you for the contribution, seems to me that you should add a new blank line after this one, as before, and to have a correct syntax in the Spanish language.

 ```suggestion
Pagas 💸.

```

**Review by realFranco — CHANGES_REQUESTED** [review]

**Review by LaurEars — APPROVED** [review]
Gracias 🙏

**Review by realFranco — APPROVED** [review]

**Review by piaria — APPROVED** [review]
Gracias!

**Comment by tiangolo** [comment]
Thank you @andresbermeoq! 🚀 

Thanks for the reviews @LaurEars, @realFranco, @piaria ☕ 

Especial thanks to @LaurEars for checking it locally and commenting that, that helps a lot! 🙇

**Pull request merged**